sdk/quantum/Azure.Quantum.Jobs/README.md

```diff
@@ -34,27 +21,43 @@ Include a section after the install command that details any requirements that m
 
 > You must have an [Azure subscription](https://azure.microsoft.com/free/), [Cosmos DB account](https://docs.microsoft.com/azure/cosmos-db/account-overview) (SQL API), and [Python 3.6+](https://www.python.org/downloads/) to use this package.
 
-### Authenticate the client
+### Authentication
```

Comment:
Changing the names of some of these sections will break DocWarden that runs as part of the CI.  I believe "Authenticate the client" is required.